packages/cookie-consent/src/CookieConsentProvider/CookieConsentProvider.tsx

```diff
@@ -0,0 +1,227 @@
+import cookie from 'cookie'
+import type { ReactNode } from 'react'
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react'
+import { uniq } from '../helpers/array'
+import { stringToHash } from '../helpers/misc'
+import type { CategoryKind, Config, Consent, Integrations } from './types'
+import { useSegmentIntegrations } from './useSegmentIntegrations'
+
+const COOKIE_PREFIX = '_scw_rgpd'
```

Comment:
Should this be customizable by config ?

Reply:
At first we could only mirror exactly what we have now and then add new features.
We don't expect for people to use this package, we only intend to make it public

But we could do some changes yes.

Personnally I would have put them as default value in order to ease the integration of this external package and limit risks